build/configs/rtl8721csm/hello/defconfig

```diff
@@ -415,6 +415,7 @@ CONFIG_ARCH_HAVE_I2CRESET=y
 # CONFIG_I2C is not set
 CONFIG_SPI=y
 CONFIG_SPI_USERIO=y
+# CONFIG_SPI_CS is not set
```

Comment:
Hello @kaikaikaime , could you please add it in some Kconfig? I think it will be difficult for the user as they cant enable it if required if added direclty in the defconfig.